expression/integration_test.go

```diff
@@ -3904,6 +3904,46 @@ func (s *testIntegrationSuite) TestAggregationBuiltinGroupConcat(c *C) {
 	tk.MustQuery("select * from d").Check(testkit.Rows("hello,h"))
 }
 
+func (s *testIntegrationSuite) TestAggregationBuiltinJSONArrayagg(c *C) {
```

Comment:
Please add more tests about `json-arrayagg distinct`.

Reply:
json_arrayagg does not support distinct in MySQL, maybe I should delete its implementation.